frontend/src/scenes/dashboard/dashboardLogic.js

```diff
@@ -396,5 +405,15 @@ export const dashboardLogic = kea({
             await breakpoint(100)
             actions.triggerDashboardUpdate({ tags: values.dashboard.tags.filter((_tag) => _tag !== tag) })
         },
+        setAutoRefresh: ({ enabled }) => {
+            if (cache.autoRefreshInterval) {
+                window.clearInterval(cache.autoRefreshInterval)
+            }
+            if (enabled) {
+                cache.autoRefreshInterval = window.setInterval(() => {
+                    actions.loadDashboardItems({ refresh: true })
+                }, AUTO_REFRESH_INTERVAL_MINS * 60 * 1000)
+            }
```

Comment:
This will cause problems if the logic is unmounted. I'd suggest adding a `beforeUnmount` event and also clearing the interval there.

Reply:
Right, thanks @mariusandra! Ready, will merge after tests pass, but feel free to re-review